pkgs/development/python-modules/pyqt/5.x.nix

```diff
@@ -103,7 +119,7 @@ in buildPythonPackage rec {
     runHook postConfigure
   '';
 
-  postInstall = ''
+  ${if (!isPy3k) then "postInstall" else null} = ''
```

Comment:
I would prefer using lib.optionalString here instead.

Reply:
Changed to `lib.optionalString`.